src/components/signup/components/SignUp.js

```diff
@@ -1,31 +1,52 @@
 import React from 'react';
-import { Card, Row, Col, Typography, Button } from 'antd';
+import { Card, Row, Col, Typography, Button, Alert } from 'antd';
+// import { Link } from 'react-router-dom';
```

Comment:
Yeah this was what I was talking about, each time one needs to point to a link, a new process.